rate-providers/registry.json

```diff
@@ -17,6 +17,25 @@
                 }
             ]
         },
+        "0x1aCB59d7c5D23C0310451bcd7bA5AE46d18c108C": {
```

Comment:
There is an "ethereum" higher level element in this file. This rate provider should be grouped into the "ethereum"